netdata-installer.sh

```diff
@@ -204,6 +210,9 @@ while [ ! -z "${1}" ]; do
 	elif [ "$1" = "--auto-update" -o "$1" = "-u" ]; then
 		AUTOUPDATE=1
 		shift 1
+	elif [ "$1" = "--stable-channel" -o "$1" = "-u" ]; then
```

Comment:
The -o "$1" = "-u" is just copy/paste error.

Reply:
fixed